product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: "Demonstration of Oracle compatible functions and syntax"
+navTitle: "Demo: Oracle compatibility"
+---
+<figure class="float-right ml-1 mb-1">
+  <iframe width="360" height="202" src="https://www.youtube.com/embed/lV4QQ53kgew" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <figcaption class="text-center font-italic">Video recording of this demonstration</figcaption>
+</figure>
+
+[EDB Postgres Advanced Server's Oracle compatible syntax and supporting packages](https://www.enterprisedb.com/docs/epas/latest/epas_compat_ora_dev_guide/) are available in BigAnimal. If you'd like to see them in action, there's a test cluster available, populated with the [Chinook sample database](https://github.com/lerocha/chinook-database): feel free to connect and try out a few queries! This topic presents a couple to get you started...
+
+
+You can use any recent version of psql to connect to EDB Postgres Advanced Server. If you choose to use the version that ships with Advanced Server, you'll get a few nice SQL\*Plus compatibility features (with more availability in [EDB\*Plus](https://www.enterprisedb.com/docs/epas/latest/edb_plus/)). The queries and commands that we'll examine here will work the same in either version of psql. For convenience, these examples use the version of psql available in Azure's Cloud Shell; you can launch this directly from the Azure portal, or on your desktop using Windows Terminal:
+
+```shell
+Welcome to Azure Cloud Shell
+
+Type "az" to use Azure CLI
+Type "help" to learn about Cloud Shell
+
+$
+```
+
+The connection string for the demo Advanced Server cluster looks like this:
+
+```
+postgres://demo:password@p-c5kdmd43ehc3kenab0c0.dqrgadbpjqv8pwzl.edbcloud.io:5432/chinook?sslmode=require
```

Comment:
I'm pretty sure edbcloud.io should be biganimal.io

Reply:
Good catch. I... Suspect I'm going to have to rebuild this to correct that though; worth holding off on release you think?

Reply:
Updated to biganimal.io (video I'll re-record tonight)